.changeset/stale-mugs-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-tech-insights-backend': patch
+---
+
+Remove unnecessary query filter
```

Comment:
you can remove the changeset since it's just a "hotfix" that goes into the pr that just got merged